survey/2022/summary.md

```diff
@@ -0,0 +1,129 @@
+# Survey Summary and Takeaways
+
+This document summarizes the survey results and gives some insight into some
+takeaways from each category of the survey.
+
+In total, we had package manager maintainers (and/or reputable sources) from 11
+different package manager ecosystems responded to the survey. Including:
+
+-   PyPI (Python
+-   Gradle (JVM)
+-   Basalt  (Bash)
+-   Opam (OCaml)
+-   RubyGems (ruby)
+-   Npm (javascript)
+-   Pkg.go.dev (go)
+-   Cargo (Rust)
+-   Pub (Dart)
+-   Apache Maven
+-   Maven Central
+
+## Integrity
+
+Most support is still around developer owned keys for signing (with PGP being
+the most supported signature type), with some a couple ecosystems with no
+support for signing yet - but there is interest/plans.
+
+There are no package managers that support keyless signing yet.
+
+Java, OCaml, Go, Ruby and Rust support reproducible builds with Maven Central
+making it mandatory.
+
+![image](https://user-images.githubusercontent.com/3060102/220680128-180c076c-31de-4f32-884e-7c2de8137dc0.png)
+
+## Typos & dependency confusion
+
+45.5% of package managers do not require and do not plan on having DNS
+verification proof when reserving/exercising a domain/namespace or domain name
+attribute of a package. This could lead to impersonation of
+organisations/entities. A method to reduce the cost of protecting against this
```

Comment:
```suggestion
organizations/entities. A method to reduce the cost of protecting against this
```
Nit: this is the en_GB spelling of organisations, but the rest of the doc is en_US.